src/beacon_metrics_gazer/beacon_metrics_gazer_launcher.star

```diff
@@ -56,14 +59,28 @@ def launch_beacon_metrics_gazer(
     )
 
 
-def get_config(beacon_http_url, node_selectors):
+def get_config(
+    beacon_http_url,
+    node_selectors,
+    port_publisher,
+    additional_service_index,
+):
     config_file_path = shared_utils.path_join(
         BEACON_METRICS_GAZER_CONFIG_MOUNT_DIRPATH_ON_SERVICE,
         BEACON_METRICS_GAZER_CONFIG_FILENAME,
     )
+
+    public_ports = shared_utils.get_additional_service_standard_public_port(
+        port_publisher,
+        constants.HTTP_PORT_ID,
+        additional_service_index,
+        0,
```

Comment:
Normally most additional services only have one public port. However, some like blockscout_launcher have two for two different configs. Since both configs use the same `additional_service_index` I need to count up the port differently. Therefore I use the index of the returned port numbers.

Checkout [this function}(https://github.com/TobiWo/ethereum-package/blob/0e8d0a4f41b6022279ee61f87e5be41f6d0c3aad/src/shared_utils/shared_utils.star#L311) which prevents code duplication quite a lot.

1. No additional service has more than two public ports
2. Therefore, I get a list with two port numbers for every additional service (defined [here](https://github.com/TobiWo/ethereum-package/blob/0e8d0a4f41b6022279ee61f87e5be41f6d0c3aad/src/shared_utils/shared_utils.star#L263)
3. Most of the time I only need one port. This is where you see the `0`
4. For some additional services, like blockscout_launcher, two public ports are needed which is why I use index `1` from the port number list as well (for the second public port, second config)

I hope this makes sense.

Reply:
As mentioned in discord, I could create these two constants:

```
ADDITIONAL_SERVICE_FIRST_PUBLIC_PORT_INDEX = 0
ADDITIONAL_SERVICE_SECOND_PUBLIC_PORT_INDEX = 1
```

and reference them in the services but I'm not sure whether this makes things clearer. Let met know what you think.

Reply:
its fine like this.